setup.py

```diff
@@ -54,7 +54,7 @@ def read(filename):
         "h5py>=3.4, <4",
         "numpy>=1.20, <2",
         "scipy>=1.1, <2",
-        "matplotlib>=2.2",
```

Comment:
Considering that there are no real changes other than the requirements, I'd personally prefer the updated reqs to just be one commit.

Also, gridspec here we gooo :D